lib/osf-components/addon/components/editable-field/license-manager/component.ts

```diff
@@ -18,18 +18,6 @@ import Analytics from 'ember-osf-web/services/analytics';
 
 import template from './template';
 
-export interface LicenseManager {
```

Comment:
🤔I don't get why this was deleted.

Reply:
I don't think this is deleted. It's moved to `registries/components/registries-license-picker`